src/app/home/superset/superset.service.ts

```diff
@@ -0,0 +1,119 @@
+/** Angular Imports */
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+/** rxjs Imports */
+import { Observable, throwError } from 'rxjs';
+import { catchError, switchMap } from 'rxjs/operators';
+
+/** EmbedDashboard SDK import */
+import { embedDashboard } from '@superset-ui/embedded-sdk';
+
+/** 
+ * Superset Service
+ */
+@Injectable({
+  providedIn: 'root'
+})
+export class SupersetService {
+  
+  /**
+   * API URL of Superset to send request
+   */
+  private apiUrl = 'http://localhost:8088/api/v1/security';
```

Comment:
Please move all of the configuration into a property file and do not hardcode it.

Reply:
Do you want me to make config.json and move the URL, username, and password into it?

Reply:
That would be better, but still... It is on UI...anybody can retrieve these information... this is not secure... i cant imagine anybody would want to share with the world the username and password....

Reply:
Hmm then we can make a service in our fineract backend to access this URL, username, and password in the frontend

Reply:
Becuase we need to send the request to some other app

Reply:
If you want my honest opinion: this feature is not really related to fineract. I would not try to combine them together... i might be wrong, but i dont see what is the benefit here... also hence the security concerns, noone would use it in production...or publicly deployed...

Reply:
okay then I will add a config.json file

Reply:
@bharathcgowda @edcable @josehernandezfintecheandomx What are your opinions about this?

Reply:
@adamsaghy @PC-11-00   I do not have a solution to suggest by but I feel it is not good to store username and password anywhere in the code or in config.json file.
It is always gets blocks during security screening.

Reply:
Yeah. that is the main reason i dont really think it is a feature to be added... :/ This looks to me more like a separate application...

Reply:
@adamsaghy Yeah that's correct apache superset is a different application, which  I am embedding into the web app , it's not a feature, to run Apache superset dashboard, we need to run Apache superset and web app separately 
You can read this blog: https://medium.com/@chaudharypushpendra.11.10.2000/embedding-of-apache-superset-dashboard-in-the-mifos-initiative-angular-web-app-b9259f1f1f1b

Reply:
You can check their git repo: https://github.com/apache/superset
And the issue: https://github.com/openMF/web-app/issues/1649 was to embed the apache superset

Reply:
so if i understand correctly, adding this dashboard into the web-app was your Gsoc assignment? Can you please share more details about what was exactly the job here? 
Nevertheless hardcoding authentication details into UI (even if it is config.json) file is wrong, and a huge security risk!

I dont really see why should the wep-app serve 2 total different application:
- Fineract backend
- Apache superset backend

Reply:
Is there any way to remove the credentials from the UI? i dont mind if the user must log into manually into the apache superset or the authentication is sync within the superset and fineract, but i just cant wrap my head around such an idea to hardcode into UI any of these sensitive informations...

Reply:
I know it is wrong to hardcode the username and password, but I asked @josehernandezfintecheandomx to make a service in fineract backend from where we can fetch the Guest Token of Apache Superset, but he denied making that
but we need to store those credentials  somewhere and  I didn't find any solution and so I hard coded.

Reply:
I have dropped a message to Ed and Victor regarding my concerns with this feature to be added to the UI